docs/configuration.md

```diff
@@ -196,6 +210,12 @@ dependencies:
   - someDep
 ```
 
+### Common problems
```

Comment:
I think this should go in the Troubleshooting page, perhaps?

Reply:
Good idea, done in 386f87cf367fe7d3ddae23fedd5c418f7b60d98b